internal/server/models/organization.go

```diff
@@ -11,7 +11,7 @@ import (
 type Organization struct {
 	Model
 
-	Name      string `gorm:"uniqueIndex:idx_organizations_name,where:deleted_at is NULL"`
+	Name      string
```

Comment:
I removed the need for this to be a unique index. We can uniquely identify orgs by their domain. They can name it as they like.